File: Python/NumberFun.py
def main():
    n = int(input())
    for i in range(0,n):
        l = input().split()
        a = int(l[0])
        b = int(l[1])
        c = int(l[2])
        if a + b == c or a * b == c or a / b == c or b / a == c or a - b == c or b - a == c:
            print("Possible")
        else:
            print("Impossible")
main()

File: C++/NumberFun.cpp
#include <iostream>

using namespace std;

int main() {
  int n;
  cin >> n;
  for (int i = 0; i < n; i++) {
    double a, b, c;
    cin >> a >> b >> c;
    if (a + b == c || a * b == c || a / b == c || b / a == c || a - b == c || b - a == c) {
      cout << "Possible" << endl;
    } else {
      cout << "Impossible" << endl;
    }
  }
}
